templates/practice_settings/general_list.html

```diff
@@ -3,33 +3,58 @@
 {php}html_header_show();{/php}
 
 <link rel="stylesheet" href="{$CSS_HEADER}" type="text/css">
+<link rel="stylesheet" type="text/css" href="{php} echo $GLOBALS['assets_static_relative']{/php}/bootstrap-3-3-4/dist/css/bootstrap.css">
+<link rel="stylesheet" type="text/css" href="{php} echo $GLOBALS['assets_static_relative']{/php}/bootstrap-sidebar-0-2-2/dist/css/sidebar.css">
 <link rel="stylesheet" type="text/css" href="{php}echo $GLOBALS['rootdir'] . '/../library/js/fancybox/jquery.fancybox-1.2.6.css';{/php}" media="screen" />
 <script type="text/javascript" src="{php}echo $GLOBALS['assets_static_relative']{/php}/jquery-min-1-3-2/index.js"></script>
+<script type="text/javascript" src="{php}echo $GLOBALS['assets_static_relative']{/php}/bootstrap-3-3-4/dist/js/bootstrap.js"></script>
+<script type="text/javascript" src="{php}echo $GLOBALS['assets_static_relative']{/php}/bootstrap-sidebar-0-2-2/dist/js/sidebar.js"></script>
 <script type="text/javascript" src="{php}echo $GLOBALS['rootdir'] . '/../library/js/common.js';{/php}"></script>
 </head>
-<body class="body_top">
+<body class="body_top" style="padding-top: 35px;">
 
-<div>
-    <b>{xl t='Practice Settings'}</b>
-</div>
+<nav class="navbar navbar-default navbar-fixed-top">
+    <div class="container-fluid">
+        <!-- Brand and toggle get grouped for better mobile display -->
+        <div class="navbar-header">
+            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+                <span class="sr-only">Toggle navigation</span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+            </button>
+            <a class="navbar-brand" href="#">{php}echo xlt('Practice Settings'){/php}</a>
```

Comment:
To see how the tab title is selected(it's a bit hokey :)), see here:
https://github.com/openemr/openemr/blob/master/interface/main/tabs/js/custom_bindings.js#L32
(in this case I think it was grabbing the first bolded term)

If set the title to be translated Practice Settings in the head section, I think you will be good.